signer/tests/lib_test.rs

```diff
@@ -307,23 +308,25 @@ fn sign_bls_transaction() {
 
     let raw_sig = transaction_sign_raw(&message, &bls_key).unwrap();
 
-    println!("{}", hex::encode(raw_sig.as_bytes()));
+    dbg!(hex::encode(raw_sig.as_bytes()));
 
     let sig = bls_signatures::Signature::from_bytes(&raw_sig.as_bytes()).expect("FIX ME");
 
     let bls_pk = bls_signatures::PublicKey::from_bytes(&bls_pubkey).unwrap();
 
-    let message_cbor = transaction_serialize(&message).expect("FIX ME");
+    let message = UnsignedMessage::try_from(&message).expect("FIX ME");
+    let message_cbor = message.marshal_cbor().expect("FIX ME");
 
-    println!("{}", hex::encode(&message_cbor));
+    dbg!(hex::encode(&message_cbor));
 
-    assert!(bls_pk.verify(sig, &message_cbor));
+    assert!(bls_pk.verify(sig, &message.to_signing_bytes()));
 }
 
 #[test]
 fn test_verify_bls_signature() {
     let test_value = common::load_test_vectors("../test_vectors/bls_signature.json").unwrap();
 
+    //TODO: update signature (what is the private key?)
```

Comment:
Yes we need to update the bls_signature.json file. It is generated by generate.rs and has the same issue regarding bls signing. It signs the all message and not only the CID.

Reply:
Actually it is `generated_test_cases.json` that is generated using the rust script and that need to be updated.

Reply:
It appeats that `generate.rs` is under `utils`, whilst thinking it is instead an example... Was that intended? Should I move it to be an example or add it to the `filecoin-signer` manifest manually as [an example](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples)

Reply:
You can move it back under `example`. It was a mistake I think during a refactoring.